io.embrace.sdk/Editor/EmbraceSDKDependencies.xml

```diff
@@ -4,9 +4,9 @@
       <repository>https://repo.maven.apache.org/maven2</repository>
       <repository>https://dl.google.com/dl/android/maven2</repository>
     </repositories>
-    <androidPackage spec="io.embrace:embrace-android-sdk:6.8.3">
+    <androidPackage spec="io.embrace:embrace-android-sdk:6.13.0">
```

Comment:
i noticed we also set the version in `io.embrace.sdk/Editor/Data/VersionsRepository.cs` file. any way we can avoid defining it in multiple locations?

Reply:
We actually tried this for awhile. We tried to have the `VersionsRepository.cs` create/update the `EmbraceSDKDependencies.xml`, however some customers ended up in an odd situation where their package was marked read-only. In these cases it could be build blocking as the file would not be generated or updated.